### DISCUSSION TITLE
react - LightGallery component breaks grid layout

### DISCUSSION THREAD

**Discussion by Chandler-Zhu** [sachinchoolur/lightGallery#1342] [Q&A] [answered]
I added LightGallery to my next.js app but it breaks the grid layout
before:
![image](https://user-images.githubusercontent.com/61914365/180247851-1e8d51e3-417c-49cb-b92b-1edfb10abfb0.png)
after:
![image](https://user-images.githubusercontent.com/61914365/180247906-fb8362cc-c160-4289-afb0-cc84e7170264.png)
repro link: https://codesandbox.io/s/compassionate-sunset-ge9c4q?file=/pages/index.tsx

**Comment by sachinchoolur** [comment, marked as answer]
You need to remove the wrapper div and add its classNames to lightGallery via `elementClassNames` prop

```js
<LightGallery
        elementClassNames="grid grid-cols-1 gap-y-10 gap-x-6 sm:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 xl:gap-x-8"
        speed={500}
      >
        {images.map((image) => (
          <BlurImage key={image.id} image={image} />
        ))}
      </LightGallery>
```

Updated demo - https://codesandbox.io/s/elastic-parm-7lj03g
